webpack/assets/javascripts/react_app/components/notifications/NotificationDropdown.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Dropdown, Glyphicon, MenuItem } from 'react-bootstrap';
+
+const NotificationDropdown = ({ links, id }) => {
+  const listLinks = links.map((link, i) => {
+    const liKey = `notification-link-${i}-${id}`;
+
+    return (
+      <MenuItem key={liKey} id={liKey} target="_blank" href={link.href}>
+        {link.title}
+      </MenuItem>
+    );
+  });
+
+  return (
+    <Dropdown className="pull-right dropdown-kebab-pf" pullRight key={id}
```

Comment:
It's ok, it was just something i thought is the responsibility of the composer (wrapper) component to organize it's children - and not for the child to be opinionated about it's location.

i suggest reading up on flexbox anyway, you may find it easier to deal with then floats